cmd/wallet-web/src/router/routes.js

```diff
@@ -60,6 +60,12 @@ export default [
       },
     ],
   },
+  {
+    path: 'oidc-share',
+    name: 'oidc-share',
+    component: load('layouts/OIDCShareLayout'),
+    meta: { requiresAuth: true, signin: true, disableCHAPI: true, isNavbarHidden: true },
+  },
```

Comment:
I think eventually we will have to create the `oidc` path with a number of children routes, same as for `waci` just above